test/testcne/graph_test.c

```diff
@@ -552,23 +545,23 @@ test_update_edges(void)
     count   = cne_node_edge_update(node_id, 0, tm->test_node[0].node.next_nodes,
                                    tm->test_node[0].node.nb_edges);
```

Comment:
Unfortunately this line throws an error in clang-format that we weren't able to fix. Attempting to follow its advice just results in a new error to change it back to this format. 

Since this line was not actually part of the changes in this pull request we wanted to move ahead anyway after spending entirely too much time trying to figure out how to fix it.

Reply:
We separated out this commit from the other test improvements so that it was more obvious that the clang format error is just due to this commit.